### ISSUE TITLE
add "complies with" term to MIOP

### ISSUE THREAD

**Issue by raissameyer** [BeBOP-OBON/miop#19]
**Term label:** complies with
**Guidance on the term:** Cite the DOI of the permit used, if that is not possible, note what system of authorization it is in compliance with.

As part of the discussion, we noted that the proposed term is different from "authorised by" (which is also not yet part of MIOP), but can work with or in parallel to it.

Note: These terms (and potentially others) are specific to the implementation of the protocol, not the protocol itself. 
Comment on the term: Please use this in the instantiation of this protocol. E.g. when you publish the protocol as part of a research publication.

Comment for MIOP development: This requires further input from groups that are from that roam of expertise (e.g. from the TK labels community)

**Comment by ndavieshi** [comment]
Yes it would be good to also create "authorized by" for when some entity has given its blessing (social) or permission (legal) to a protocol and/or an instantiation of it in a particular place/jurisdiction. In that case, it should link to the authorization document (DOI)

**Comment by jodievandekamp** [comment]
An example of free text used for a voyage conducted on CSIRO RV Investigator - IN2019_v03 110o Indian Ocean

In Voyage Plan under standard "Permit" section (note authorizing or compliance body not always stated and permit identifier not always used):
Permits
• No foreign clearances required – intend to finish 110°E line at ~11°30’S just south of Indonesian EEZ.
• All 110°E line stations are outside EEZ except stations at 24°30’S 110°E, 26°S 110°E & 27°30’S 110°E which 
are inside western extent of the Abrolhos Marine Park.
• Training station on 15 May in EEZ will be outside SW Corner Marine Park waters.
• Commonwealth Marine Parks permits (Beckley PA2018-00065F obtained for SW and NW Marine parks)
• Commonwealth Dept Environment permit for sampling for genetics in EEZ (Beckley AU-COM2019-446)
• Commonwealth AFMA permit for phytoplankton, zooplankton & fish larvae in EEZ (Beckley 1004152
obtained)
• Quarantine & import requirements: Samples in 10% formalin, 4% formaldehyde, 1% Lugols, 2% 
glutaraldehyde (2%) or 70% ethanol do not require permit but need to be properly documented by PIs. For 
other material (especially frozen material) Ostrowski, Raes, Seymour, Jeffs & Beckley will use institutional 
permits. Permits for Landry & Antoine in progress.
• Murdoch University Animal Ethics permit for catching zooplankton & larval fishes - Beckley R2885/16 #315
(Pelagic ecology of SE Indian Ocean) obtained
• Permit for laser (Jeffs). Federal & MNF approvals confirmed and H&S training completed.
• Permits for using radioactive sources 14C; 15N

In Acknowledgements section of Raes et al., 2022 (https://doi.org/10.1016/j.dsr2.2022.105097):
Water and plankton samples at 29–23°S, 110 °E were taken under Australian Fisheries Management Authority scientific permit 1004152, Australian Government permit AU-COM2019-446 and those in the Abrolhos Marine Park under permit PA2018-00065-1 issued by the Director of National Parks, Australia. The views expressed in this publication do not necessarily represent those of the Director of National Parks, CSIRO, or the Australian Government.

**Comment by jodievandekamp** [comment]
AFMA = Australian Fisheries Management Authority

**Comment by jodievandekamp** [comment]
The above example is quite a good one as there was a lot happening on that voyage across a vast area. Linking to the voyage landing page here:
https://www.cmar.csiro.au/data/trawler/survey_details.cfm?survey=IN2019_V03

**Comment by ndavieshi** [comment]
We should start a "package" related to "compliances and endorsements" as this is an issue (ethical, legal, social) that probably should be "minimum" for any protocol (to indicate what, if anything, the protocol relates to in this context). There is complexity over what terms to use for different types of relationship (e.g., author thinks the protocol complies with an named authority, versus the protocol has been specifically authorized by that authority). Are there terms for this we could use from elsewhere?
